src/main/java/org/jenkinsci/plugins/gravatar/factory/GravatarFactory.java

```diff
@@ -19,11 +19,6 @@ public Gravatar testGravatar() {
     }
 
     private Gravatar gravatar() {
-        return new Gravatar().setHttps(useHttps());
```

Comment:
noticed when I develop locally I was getting an http url which didn't seem right, I can't see why you would not want to fetch this over https

Reply:
I didn't change it for tests yet. you'll now need to fix the failing test to get this merged